src/system/Form/RadioBoxGroup.js

```diff
@@ -94,16 +95,65 @@ const RadioOption = ( {
 	);
 };
 
-RadioOption.propTypes = {
+const ChipOption = ( {
+	defaultValue,
+	option: { value, label },
+	name,
+	disabled,
+	onChangeHandler,
+} ) => {
+	const checked = `${ defaultValue }` === `${ value }`;
+
+	return (
+		<Button
```

Comment:
We should use the idea of an actual radio component here. To match the design, you could create the same effect using before/after pseudo-elements but still utilizing an input radio. You can check the original RadioBoxGroup for example.

Reply:
The issue is with keyboard navigation. If you are using radio, then you cannot focus an option with keyboard without making that option "active". But that is a necessary use-case for us.

Reply:
@ aswasif007 I understand your concern, but how you use the component does not define the component structure.

You must understand that people using the keyboard will need to use it the way you believe it's "wrong." The key to accessibility here is to provide a Radio option to customers, where they can use the tool with the keyboard. 

Suppose this component's `onChange` event will trigger a page render that depends on the code using this component. In the same way, using the keyboard can trigger the page render; a person clicking with the mouse left and right will do the same.

The UI describes a toggle situation, and we must provide an accessible switch pattern or, in this case, use a regular input radio pattern.

Reply:
Fair enough. Updated.